js/src/components/paid-ads/asset-group/faqs-section.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Section from '.~/wcdl/section';
+import FaqsPanel from '.~/components/faqs-panel';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+
+const faqItems = [
+	{
+		trackId: 'what-will-my-ads-look-like',
+		question: __(
+			'What will my ads look like?',
+			'google-listings-and-ads'
+		),
+		answer: (
+			<div>
+				{ createInterpolateElement(
+					__(
+						'Google will generate text ads and responsive display ads in various combinations and formats from the headlines, images, and descriptions you add. Your ads will automatically adjust their size, appearance, and format to fit available ad spaces. <link>See common ad formats</link>',
+						'google-listings-and-ads'
+					),
+					{
+						link: (
+							<AppDocumentationLink
+								context="assets-faq"
+								linkId="assets-faq-about-ad-formats-available-in-different-campaign-types"
+								href="https://support.google.com/google-ads/answer/1722124"
+							/>
+						),
+					}
+				) }
+			</div>
+		),
+	},
+	{
+		trackId: 'what-makes-these-ads-different-from-product-ads',
+		question: __(
+			'What makes these ads different from product ads?',
+			'google-listings-and-ads'
+		),
+		answer: (
+			<>
+				<div>
+					{ __(
+						`Dynamic ad assets can elevate your campaign by offering a variety of ad combinations that capture your audience's attention and generate maximum engagement. By leveraging Google's asset-mixing technology, your ads can be optimized to deliver the right message, to the right people, at the right time.`,
+						'google-listings-and-ads'
+					) }
+				</div>
+				<div>
+					{ __(
+						'Compared to product ads—which showcase individual products and are designed to drive direct sales and revenue—responsive ads with dynamic ad assets are typically used to highlight your business, generate interest, and attract new customers. While both types of ads can drive conversions, using them together can generate even greater results.',
+						'google-listings-and-ads'
+					) }
+				</div>
+			</>
+		),
+	},
+];
+
+/**
+ * Renders a toggleable FAQs section about the campaign assets of the Google Ads.
+ *
+ * @fires gla_faq with `{ context: 'campaign-management', id: 'what-will-my-ads-look-like', action: 'expand' }`.
```

Comment:
It not correct to merge both events and do something like.

`@fires gla_faq with { context: 'campaign-management', id: 'what-will-my-ads-look-like', action: 'expand|collapse'}`.

Not sure where but I think we used that way before.

Reply:
This is an interesting question. I haven't noticed before but only copied them from the other place and changed the context and id.

After viewing the src/Tracking/README.md, yes, there are a few `@fires` tags that use `|` for indicating the possible values. In addition, the `@event` tag of gla_faq already stated the possible values of `action`, so I guess it should be fine to merge them into a single one by using `|` or even omit the `action` in the `@fires` description.

![2023-02-17 18 15 54](https://user-images.githubusercontent.com/17420811/219616876-c8191d7a-80db-4ca9-ad15-493d316349b0.png)

Reply:
Merged in https://github.com/woocommerce/google-listings-and-ads/pull/1881/commits/72353d63abd3c4a6fa5b19dfd90922533cd2059b.